src/main/java/net/minecraftforge/lex/yunomakegoodmap/GuiCustomizeWorld.java

```diff
@@ -221,17 +222,36 @@ private StructInfo getInfo(ResourceLocation res)
 
         try
         {
-            IResource ir = this.mc.getResourceManager().getResource(new ResourceLocation(res.getResourceDomain(), "structures/" + res.getResourcePath() + ".json"));
-            ret = GSON.fromJson(new InputStreamReader(ir.getInputStream()), StructInfo.class);
+            if (res.getResourceDomain() == "/config/")
```

Comment:
Duplicating code is not the way to go about this, all you're editing is two input streams, which is 4 lines of code tops. Also i'll have to look into it, i've tested this specific setup before pushing... i'll have to see what went wrong.